app/celery/letters_pdf_tasks.py

```diff
@@ -207,19 +211,38 @@ def process_virus_scan_passed(self, filename):
 
     current_app.logger.info('notification id {} ({}) sanitised and ready to send'.format(notification.id, filename))
 
+    # temporarily upload original pdf while testing sanitise flow.
     _upload_pdf_to_test_or_live_pdf_bucket(
-        new_pdf,
+        old_pdf,  # TODO: change to new_pdf
         filename,
         is_test_letter=is_test_key)
 
+    billable_units = _get_page_count(notification, old_pdf)
```

Comment:
i think we should put this above the sanitise command - if the file is invalid, then there's no point making the rest request to template preview